### PR TITLE
chore: Update email copy for invite_with_enroll

### PR DESCRIPTION
**Pull request by tannerwelsh**
Last part of https://trello.com/c/yAhABcIb/957-update-reject-accept-email-copy

Related PR: https://github.com/turingschool/apply/pull/673

NB: I was running into errors getting census set up locally, and debugging my development environment for a copy-change PR didn't seem like the best use of time. I can try testing this out in staging to make sure the template works

### CODE REVIEW

File: app/views/invitation_mailer/invite_with_enroll.html.erb [hunk 1]
```diff
@@ -3,28 +3,39 @@
...
   </p>
   <p>
-  <strong>Next, please visit <a href='https://enroll.turing.io/'>enroll.turing.io</a></strong> to view enrollment availability, pay your deposit, and pick a starting date.
+    <strong>Your next steps:</strong>
   </p>
+  <ol>
+    <li>
+      <strong>Follow <%= link_to("this link", "http://email.turing.io/wf/click?upn=ZphGLUw808sdEm1ILwvOtBRVLCxNVRDaqd7qDnM2nWhs-2FsFHFqP3treA9dqm6BfXDnSJc1m-2FjQhnupW0Lksbc9Pk3GeP3y5-2BIgrmHpr7cS-2FSqyXFk1HlKAU6WL0OrdD15jddPCYLAzXn2nhdSbruGwvT28l808SlqijM20asKN4-3D_et-2FIvjrxoz0bIAuQSC0hCSSIWpYW1K-2BlkFEivNtVabwVpn03BYLm9Nit1Rv4uSdUDGyAEzg3JfR3IsacBG8lcxpGMXzPrxXbhfyYTAiCncvC73J6fWIOGpc9FGaGLpQaWBeragOfo3gJSfvvAIfOmew38h4-2ByWUWgGZCAZOgMFJI24sJeIp1iud5eZ87D6CllREhlczZhUXc4PudTHHgIw-3D-3D") %></strong> to create an account with Census, the Turing School of Software & Design's identity management application.
```
Comment (kat3kasper) [inline app/views/invitation_mailer/invite_with_enroll.html.erb:16]: This is a custom jwt link created by Census to allow a user to create an account.

```suggestion
      <strong>Follow <%= link_to("this link", @url) %></strong> to create an account with Census, the Turing School of Software & Design's identity management application.
```

File: app/views/invitation_mailer/invite_with_enroll.html.erb [hunk 2]
```diff
@@ -3,28 +3,39 @@
...
+    </li>
+    <li>
+      <strong>Visit <%= link_to("enroll.turing.io", "http://email.turing.io/wf/click?upn=eZSed92pY-2FQyfwSCgLIfgUmJoM64eS3COuXPA5GJ8ho-3D_et-2FIvjrxoz0bIAuQSC0hCSSIWpYW1K-2BlkFEivNtVabwVpn03BYLm9Nit1Rv4uSdUwK-2BQFhmeEHRO2ehUYlapgkgZ935y3lYYKeMLokTsl2yXOqZY-2BftDHpuvq5FZgEWAUGCenVFiG9qercaHQXfuSkkgT315JX-2B9CJ1UeA5-2FCsPjRarpwqtiNmakDkMdSzxRGqYjYop8DefTZZdN2bNaKA-3D-3D") %></strong> to pay your deposit, choose a Mod 0, and choose a cohort. We recommend you do this ASAP, as cohorts fill up quickly.
```
Comment (kat3kasper) [inline app/views/invitation_mailer/invite_with_enroll.html.erb:19]:
```suggestion
      <strong>Visit <%= link_to("enroll.turing.io", "https://enroll.turing.io/") %></strong> to pay your deposit, choose a Mod 0, and choose a cohort. We recommend you do this ASAP, as cohorts fill up quickly. 
```

File: app/views/invitation_mailer/invite_with_enroll.html.erb [hunk 3]
```diff
@@ -3,28 +3,39 @@
...
+    </li>
+  </ol>
   <p>We invite you to explore the following financing options available to our students:</p>
   <ul>
-    <li><%= link_to("Sallie Mae", "https://www.salliemae.com/ctsmartloan") %></li>
-    <li><%= link_to("SkillsFund", "http://turing.skills.fund/accepted") %></li>
-    <li><%= link_to("Climb",      "http://www.climbcredit.com/apply/turing") %></li>
+    <li><%= link_to("Sallie Mae", "http://email.turing.io/wf/click?upn=GlpLQqCoibpxXvPZd6HJXQpWC1UAvFGACTTOo0Ur4t8Olvp7S4m43GWwMMV4gErP_et-2FIvjrxoz0bIAuQSC0hCSSIWpYW1K-2BlkFEivNtVabwVpn03BYLm9Nit1Rv4uSdUrnfyyfj4xqMi1YKu4d9RsOVdJwbL-2F2Wcdx8EFcOnjR0-2FA27lHcCkdOc-2BN93mNwJ-2FAOm1l7z6l-2F4Se-2B5EGiCo-2FRt-2BPSiiUR9-2Bpb8r9R4SOqN6VQdSkdmHlPzcqHx62v4AF-2BBhEcxg-2Fm3iT8L1uIdMAQ-3D-3D") %></li>
```
Comment (kat3kasper) [inline app/views/invitation_mailer/invite_with_enroll.html.erb:24]:
```suggestion
    <li><%= link_to("Sallie Mae", "https://www.salliemae.com/ctsmartloan") %></li>
```

File: app/views/invitation_mailer/invite_with_enroll.html.erb [hunk 4]
```diff
@@ -3,28 +3,39 @@
...
+    <li><%= link_to("SkillsFund", "http://email.turing.io/wf/click?upn=x9jGS5kbtoI77qbqIokdVQSiwp700oFIOhRWODv8yj8WwluZMcv-2B3pwvYr1ZPcO3_et-2FIvjrxoz0bIAuQSC0hCSSIWpYW1K-2BlkFEivNtVabwVpn03BYLm9Nit1Rv4uSdU9w99c7yGIEKHQjdNB3O-2F6HhC87-2BKw7mwEpJths02xrFF-2BAZ0Bsrl9DLQd4JAQHrkM9lMA48eH4gIPU4unem58sBmMr2bDqKKqucDe7oHHaPn4FaO-2FgssbwGUbBsxWhd51nBR-2B3-2Bbu-2FcFUxLnNj2RGA-3D-3D") %></li>
```
Comment (kat3kasper) [inline app/views/invitation_mailer/invite_with_enroll.html.erb:25]:
```suggestion
    <li><%= link_to("SkillsFund", "https://turing.skills.fund/") %></li>
```

File: app/views/invitation_mailer/invite_with_enroll.html.erb [hunk 5]
```diff
@@ -3,28 +3,39 @@
...
+    <li><%= link_to("Climb",      "http://email.turing.io/wf/click?upn=lUiR9f3M7RhCH6WaTBXRhUXP-2FUA2dfhnzpz617MYp21VNCXtJiqkRfrMBqVxDi4B_et-2FIvjrxoz0bIAuQSC0hCSSIWpYW1K-2BlkFEivNtVabwVpn03BYLm9Nit1Rv4uSdURzaiagUINGr3ph129NFh1-2BXoOsdSYPyB25OvmEgLO4k0xBQTJrrzaeAEdtmRpQW7C6-2FrfXAV5Z1CPuSclMvA7Kpy9FnF9p-2FcxPePfwO1Z8djUSLuvFNziitkFe8ek8nFUxqzyZ2-2FTc86S1WthW-2BzBA-3D-3D") %></li>
```
Comment (kat3kasper) [inline app/views/invitation_mailer/invite_with_enroll.html.erb:26]:
```suggestion
    <li><%= link_to("Climb", "https://climbcredit.com/apply/turing") %></li>
```

**Review by kat3kasper — CHANGES_REQUESTED** [review]
I am guessing most of the links provided in the google doc were copy and pasted from a census email where the links were replaced by the email service. I believe we want to keep the original links for most of them rather than the formatted click tracking links

**Comment by tannerwelsh** [comment]
> I am guessing most of the links provided in the google doc were copy and pasted from a census email where the links were replaced by the email service. I believe we want to keep the original links for most of them rather than the formatted click tracking links

Got it, thanks. I thought the email tracking links were intentional, but that makes more sense.

**Review by kat3kasper — APPROVED** [review]
Thanks for updating this! Should be GTM

**Pull request merged**